test/control-plane/services/dual-stack/lbmap1.golden

```diff
@@ -0,0 +1,36 @@
+ - Services -------------------------------------------------------------------------
```

Comment:
This is the expected output for a particular k8s versions, should we have different lbmap1.golden files depending on the k8s version? Probably it could be a symlinks pointing to the same output and if one of them differs then it wouldn't be a symlink.

Reply:
Done.